vc14 syntax error for empty array bladerf1.c

master branch - not sure how to fix this. Its an annoying compiler inconsistency, it doesn't seem to support size zero static arrays caused by the `#if 0`

Source\bladeRF\host\libraries\libbladeRF\src\board\bladerf1\bladerf1.c(371): error C2059: syntax error: '}' [Build\bladeRF\libraries\libbladeRF\libbladerf_shared.vcxproj]

```
static const struct bladerf_lms_port_name_map bladerf1_tx_port_map[] = {
/* TODO: TX PA control is not implemented */
#if 0
    {
        FIELD_INIT(.name, "aux"),
        FIELD_INIT(.tx_pa, PA_AUX),
    },
    {
        FIELD_INIT(.name, "pa1"),
        FIELD_INIT(.tx_pa, PA_1),
    },
    {
        FIELD_INIT(.name, "pa2"),
        FIELD_INIT(.tx_pa, PA_2),
    },
    {
        FIELD_INIT(.name, "none"),
        FIELD_INIT(.tx_pa, PA_NONE),
    },
#endif  // 0
};
```

--- a/host/libraries/libbladeRF/src/board/bladerf1/bladerf1.c
+++ b/host/libraries/libbladeRF/src/board/bladerf1/bladerf1.c
@@ -349,8 +349,6 @@ static const struct bladerf_lms_port_name_map bladerf1_rx_port_map[] = {
 };
 
 static const struct bladerf_lms_port_name_map bladerf1_tx_port_map[] = {
-/* TODO: TX PA control is not implemented */
-#if 0
     {
         FIELD_INIT(.name, "aux"),
         FIELD_INIT(.tx_pa, PA_AUX),
@@ -367,7 +365,6 @@ static const struct bladerf_lms_port_name_map bladerf1_tx_port_map[] = {
         FIELD_INIT(.name, "none"),
         FIELD_INIT(.tx_pa, PA_NONE),
     },
-#endif  // 0
 };
 
 /******************************************************************************/
@@ -2178,8 +2175,9 @@ static int bladerf1_get_rf_ports(struct bladerf *dev,
     size_t i;
 
     if (BLADERF_CHANNEL_IS_TX(ch)) {
-        port_map     = bladerf1_tx_port_map;
-        port_map_len = ARRAY_SIZE(bladerf1_tx_port_map);
+        /* Return a null list instead of bladerf1_tx_port_map */
+        port_map     = NULL;
+        port_map_len = 0;
     } else {
         port_map     = bladerf1_rx_port_map;
         port_map_len = ARRAY_SIZE(bladerf1_rx_port_map);

